sycl/include/CL/sycl/handler.hpp

```diff
@@ -884,6 +885,14 @@ class __SYCL_EXPORT handler {
     MCGType = detail::CG::CODEPLAY_HOST_TASK;
   }
 
+  template <typename FuncT>
+  typename std::enable_if<
+      detail::check_fn_signature<typename std::remove_reference<FuncT>::type,
+                                 void(interop_handle)>::value>::type
+  codeplay_host_task(FuncT) {
+    throw runtime_error("Not implemented", PI_ERROR_UNKNOWN);
```

Comment:
Would prefer a more specific error message. "codeplay_host_task not implemented", maybe?

Reply:
This test is removed.